langchain/vectorstores/chroma.py

```diff
@@ -356,7 +356,7 @@ def update_document(self, document_id: str, document: Document) -> None:
             raise ValueError(
                 "For update, you must specify an embedding function on creation."
             )
-        embeddings = self._embedding_function.embed_documents(list(text))
+        embeddings = self._embedding_function.embed_documents([text])
 
         self._collection.update(
             ids=[document_id],
```

Comment:
while we're at it could we change L363 to `embeddings=embeddings`? think that would've potentially caught this earlier as well (would've seen that embeddings had different length than texts)